src/fights.ts

```diff
@@ -2247,8 +2248,24 @@ function yachtzee(): void {
 
       if (!equippedOutfit || !success()) return;
 
-      setChoice(918, 2);
+      const lastUMDDate = property.getString("umdLastObtained");
```

Comment:
I generally prefer using `get` with a default value, but this is fine

Reply:
What the heck does this property normally look like?

Reply:
`umdLastObtained=2022-05-14` is my current pref, but I assume it should otherwise be just `umdLastObtained=`. Tbh I wasn't particularly sure of the return type of `get()` on the latter, so I wanted to just force it to return as a string (this was relevant before I made the `!lastUMDDate` falsy change, not sure if it is as relevant now).